package.json

```diff
@@ -18,5 +18,8 @@
     "bugs": {
         "url": "https://github.com/FACxBeamery/grads-final-project/issues"
     },
-    "homepage": "https://github.com/FACxBeamery/grads-final-project#readme"
+    "homepage": "https://github.com/FACxBeamery/grads-final-project#readme",
+    "dependencies": {
```

Comment:
https://github.com/FACxBeamery/grads-final-project/pull/4/commits/52772cee1bfb10c004e1d47882013f0494e8f50a

please see this commit and install the npm package